ui/src/addCriteria.tsx

```diff
@@ -0,0 +1,202 @@
+import Box from "@mui/material/Box";
```

Comment:
optional, but we tend to use
```
import { Box, Button, Chip } from "@mui/material"
```
style imports in Axon (Enterprise Terra UI)

you seem to have a mix here

Reply:
We've gone with the individual imports based on the docs saying in can decrease startup time in development. It looks like there's another option to enable that more concise syntax using a Babel plugin so we could explore that at some point.

For now, I've changed the one instance using multiple imports to be consistent the others.